src/Stratis.Bitcoin.Tests/Consensus/ChainedHeaderTreeTest.cs

```diff
@@ -1966,6 +2030,122 @@ public void ChainWith10DataAvailableHeaders_PartialValidationSucceededOnTheFirst
             listOfHeaders.Should().HaveCount(1);
             listOfHeaders.First().ChainedHeader.HashBlock.Should().Be(secondHeader.HashBlock);
         }
+      
+        /// <summary>
+        /// Issue 19 @ New peer K claims 10d. Peer K disconnects. Chain shouldn't change.
+        /// </summary>
+        [Fact]
+        public void NewPeerClaimsHeadSecondTime_PeerDisconnected_ChainShouldNotChange()
+        {
+            const int initialChainSize = 5;
+            TestContext ctx = new TestContextBuilder().WithInitialChain(initialChainSize).UseCheckpoints().Build();
+            ChainedHeaderTree cht = ctx.ChainedHeaderTree;
+            ChainedHeader initialChainTip = ctx.InitialChainTip;
+
+            IEnumerable<BlockHeader> listOfAllUniqueBlockHeaders = ctx.SetupPeersForTest(initialChainSize, ctx, cht, initialChainTip, out ChainedHeader chainATip, out ChainedHeader chainDTip);
+            CheckChainedHeaderTreeConsistency(cht, listOfAllUniqueBlockHeaders);
+
+            //Additional SetUp for current test
```

Comment:
Use space after slashes, everywhere. Use "." At the end of the sentence, everywhere.

Reply:
ok